contrib/lib/tests/templates.rs

```diff
@@ -33,11 +33,12 @@ mod templates_tests {
     }
 
     #[test]
+    #[allow(clippy::try_err)]
```

Comment:
There is [an issue](https://github.com/rust-lang/rust-clippy/issues/5757) to disable this lint for try blocks. Since the [try_err](https://rust-lang.github.io/rust-clippy/master/index.html#try_err) lint purpose is to avoid this style for early return, I think we can ask them to ignore function return for this lint.

What do you think?